crates/bevy_render/src/view/mod.rs

```diff
@@ -794,11 +806,32 @@ struct MainTargetTextures {
     main_texture: Arc<AtomicUsize>,
 }
 
-#[allow(clippy::too_many_arguments)]
-pub fn prepare_view_targets(
-    mut commands: Commands,
+pub fn prepare_view_textures(
```

Comment:
Nit: Very brief doc comments for both of thees systems would also be nice if you feel up to it.